app/src/main/java/com/fidesmo/sec/servicedeliverydemo/ServiceDeliveryActivity.java

```diff
@@ -87,7 +88,7 @@ void callFidesmoApp(String action, String spId, String serviceId) {
         try {
             // create Intent to one of the two Actions exposed by the Fidesmo
             // App
-            Intent intent = new Intent(action, Uri.parse(SERVICE_URI + spId + "/" + serviceId));
+            Intent intent = new Intent(action, Uri.parse(SERVICE_URI + spId + "/services/" + serviceId));
```

Comment:
it was `https://api.fidesmo.com/service/appId_xxxx/serviceId_xxxx ()`
but now it is `https://api.fidesmo.com/appId_xxxx/services/serviceId_xxxx`, maybe to follow the same format as in web. I'm just follow what is implemented inside the app:

https://github.com/fidesmo/android-sec-client/blob/master/src/main/java/com/fidesmo/sec/android/application/AppActivity.kt#L180-L186